server/sonar-server/src/main/java/org/sonar/server/component/ws/AppAction.java

```diff
@@ -181,12 +205,6 @@ private ComponentDto retrieveRootIfNotCurrentComponent(ComponentDto componentDto
   }
 
   @CheckForNull
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getRootUuid"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)